### ISSUE TITLE
InvalidSlotTypeName

### ISSUE THREAD

**Issue by xynazog** [alexa/skill-sample-nodejs-salesforce#3]
I am getting the following error while using the `ask deploy` for my application.
```
-------------------- Update Skill Project --------------------
Skill Id: <hidden>
Skill deployment finished.
Call update-model error.
Error code: 400
{
  "message": "Interaction model is not valid.",
  "violations": [
    {
      "message": "InvalidSlotTypeName: Slot type name \"AMAZON.SearchQuery\" is invalid. The name of a custom slot type must begin with an alphabetic character and can consist only of alphabetic characters or underscores."
    }
  ]
}
```

Is this something related to the code in the repo? I have been trying to fix the issue using the JSON editor on my Alexa Skills dashboard but it won't work. I found a similar problem on the [Salesforce forum](https://developer.salesforce.com/forums/?id=9060G0000005dL4QAI) but it has no solution.

**Comment by Torsen** [comment]
Replace all the "AMAZON.SearchQuery" for "AMAZONSearchQuery" in the "en-US.json" file

**Comment by ericchenmou** [comment]
I made a fix for this in commit [bc5aced63e75b41a06c32b8f63ae11c26b2ba2c1](https://github.com/alexa/skill-sample-nodejs-salesforce/commit/bc5aced63e75b41a06c32b8f63ae11c26b2ba2c1) - tagged to the wrong issue in the commit. Please give it a shot and let me know if you are still seeing any issues.

**Comment by xynazog** [comment]
@ericchenmou Thanks a lot! 💯
